Code/Framework/AzFramework/Platform/Windows/AzFramework/Input/Devices/Gamepad/InputDeviceGamepad_Windows.h

```diff
@@ -0,0 +1,81 @@
+    class InputDeviceGamepadWindows : public InputDeviceGamepad::Implementation
+                                    , public RawInputNotificationBusWindows::Handler
+    {
+    public:
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        // Allocator
+        AZ_CLASS_ALLOCATOR(InputDeviceGamepadWindows, AZ::SystemAllocator);
+
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        //! Constructor
+        //! \param[in] inputDevice Reference to the input device being implemented
+        //! \param[in] xinputModuleHandle Shared pointer to the xinput dynamic module
+        InputDeviceGamepadWindows(InputDeviceGamepad& inputDevice,
+                                  AZStd::shared_ptr<AZ::DynamicModuleHandle> xinputModuleHandle);
+
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        //! Destructor
+        ~InputDeviceGamepadWindows() override;
+
+    private:
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        //! \ref AzFramework::InputDeviceGamepad::Implementation::IsConnected
+        bool IsConnected() const override;
+
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        //! \ref AzFramework::InputDeviceGamepad::Implementation::SetVibration
+        void SetVibration(float leftMotorSpeedNormalized, float rightMotorSpeedNormalized) override;
+
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        //! \ref AzFramework::InputDeviceGamepad::Implementation::GetPhysicalKeyOrButtonText
+        bool GetPhysicalKeyOrButtonText(const InputChannelId& inputChannelId,
+                                        AZStd::string& o_keyOrButtonText) const override;
+
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        //! \ref AzFramework::InputDeviceGamepad::Implementation::TickInputDevice
+        void TickInputDevice() override;
+
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        //! \ref AzFramework::RawInputNotificationsWindows::OnRawInputDeviceChangeEvent
+        void OnRawInputDeviceChangeEvent() override;
+
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        // Variables
+        AZStd::shared_ptr<AZ::DynamicModuleHandle> m_xinputModuleHandle; //!< Handle to the xinput module
+        RawGamepadState                            m_rawGamepadState;    //!< The last known raw game-pad state
+        bool                                       m_isConnected;        //!< Is this game-pad currently connected?
+        bool                                       m_tryConnect;         //!< Check whether this game-pad just connected?
```

Comment:
Can you add curly braces here to make sure these variables are always zero-intialized even if a new constructor is added in the future?

That will set bools to `false` and enum values to `0`
```suggestion
        RawGamepadState                            m_rawGamepadState{};    //!< The last known raw game-pad state
        bool                                       m_isConnected{};        //!< Is this game-pad currently connected?
        bool                                       m_tryConnect{};         //!< Check whether this game-pad just connected?
```